### ISSUE TITLE
Can sharpfuzz support binary-only(no-source code) fuzzing?

### ISSUE THREAD

**Issue by WuHanDuanTao** [Metalnem/sharpfuzz#64]
When source code is NOT available, we only have the compiled binary file, does sharpfuzz support this scenario?

**Comment by mschessler** [comment]
Since SharpFuzz instruments the code on IL level there should not be any problem getting it to work.

**Comment by Metalnem** [comment]
Yep, @mschessler is correct. In fact, SharpFuzz supports *only* compiled binary files.
